### DISCUSSION TITLE
V0.2.5 Commodore PC10-III, with FE2010A and NEC V20 - BIOS has problem seeing the V20?!?

### DISCUSSION THREAD

**Discussion by Skynet1HA** [640-KB/GLaBIOS#58] [Tech Support]
Hi there.

When i try the FE2010A version on my commodore PC10-III with the V20 in it, the BIOS fails to see that is it a V20 there is in the computer for some reason.
I get beep codes saying: "V20 not installed. Replace with the 8088 version of GLaBIOS ROM" ?!? 

Other bios i have tried, do see it as a V20.

I dont really know how to get there, but my own final goal is to get the computer to initialize my 8/16 bit VGA card, instead of the internal CGA/EGA.
- and this bios looked promishing, as i could see it would run V20 and also VGA on some machines. :-)

**Comment by Skynet1HA** [comment]
hmmm...

It might be a sequence of "4 long and 1 short", "Base RAM (0-16KB) parity error", but that dont make any sense to me, as i have no problem with memory on any other bios or test i run?!?

But now i have tried both with a Siemens 8088 and that corosponding GLABIOS_0.2.5_8F.rom, and a NEC V20 with GLABIOS_0.2.5_VF.ROM, and i am getting the same beep-codes anyway...

I have no onboard ram thoug, but i run a Monotech PC MicroRAM in an 8bit slot instead...
- and it works fine on anything else until now (if that is the problem)

**Comment by 640-KB** [comment]
Hey there.  So you're confident it is 4 long and 1 short?  If so, yeah that's a parity/NMI flag, which is a pretty definitive hardware flag.  That board originally didn't have parity, right?  In which case, perhaps the original BIOS ignored parity errors entirely (since it wasn't implemented on hardware).

I've got more questions and thoughts... but first [here's a build](https://github.com/640-KB/GLaBIOS/releases/download/v0.2.5/GLABIOS_0.2.6_8F_122624_NO_PAR_1.ROM) that disables parity checking.  Let's see if that gets this up and booting and we can figure out the rest from there!

640KB

**Comment by Skynet1HA** [comment]
Hi.

Thank you for your help.

That one got me into boot the system with the onboard CGA card.
- no matter what i do, i cant boot it to my VGA card.

This bios also say:
LPT none
COM none
- but both are pressent with the commodore bios.

But some progress is there :-)

Kind Regards
Peter
 .


Den tors. 26. dec. 2024 kl. 19.32 skrev 640KB ***@***.***>:

**Comment by 640-KB** [comment]
So is [this the board](https://theretroweb.com/motherboards/s/commodore-pc-10-iii) you have?  If so, can you try changing DIP switches 3 and 4 ([page 12 of the manual](https://theretroweb.com/motherboard/manual/commodore-pc-10-iii-technical-manual-65846133c4eea755402101.pdf)) to `DOWN(ON) / DOWN(ON) NO MONITOR`, and perhaps switch 2 to UP(OFF) for mono?  

See if that works for the VGA... then we can tackle the COM/LPT ports!
